src/main/java/edu/kis/vh/nursery/factory/StackRhymersFactory.java

```diff
@@ -7,9 +7,14 @@
 import edu.kis.vh.nursery.dataStructures.IntArrayStack;
 
 public class StackRhymersFactory implements Rhymersfactory {
+    IntArrayStack stack;
+
+    public StackRhymersFactory() {
+        stack = new IntArrayStack();
+    }
+
```

Comment:
wrong approach. Different rhymer instances shouldn't share the same data structure instance